<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pathfinder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>PathfinderIcon.png</normaloff>PathfinderIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="styleSheet">
       <string notr="true">QSplitter::handle:horizontal {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
		stop:0 rgba(255, 255, 255, 0), 
		stop:0.407273 rgba(200, 200, 200, 255), 
		stop:0.4825 rgba(101, 104, 113, 235), 
		stop:0.6 rgba(255, 255, 255, 0));
    image: url(:/images/splitter.png);
	margin: 2px 0x;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="verticalSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSplitter::handle:vertical {
	background-color: qlineargradient(spread:pad, y1:0, x1:0, y2:1, x2:0, 
		stop:0 rgba(255, 255, 255, 0), 
		stop:0.407273 rgba(200, 200, 200, 255), 
		stop:0.4825 rgba(101, 104, 113, 235), 
		stop:0.6 rgba(255, 255, 255, 0));
    image: url(:/images/splitter.png);
	margin: 0px 2px;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Software List</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QLineEdit" name="softwareSearchLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>Search for software</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="softwareTreeWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>222</width>
                <height>100</height>
               </size>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="headerDefaultSectionSize">
               <number>120</number>
              </attribute>
              <column>
               <property name="text">
                <string>Header</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Item</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>License List</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QLineEdit" name="licenseSearchLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>Search for license</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="licenseListWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>222</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addSoftwareButton">
             <property name="text">
              <string>Add Software</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeSoftwareButton">
             <property name="text">
              <string>Remove Software</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="licenseTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="licenseTabWidgetPage1">
        <attribute name="title">
         <string>License</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="licensePlainTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>350</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="parseLicenseButton">
           <property name="text">
            <string>Analyze License</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Scan</string>
    </property>
    <widget class="QMenu" name="menuScan_in_drive">
     <property name="title">
      <string>Scan in drive</string>
     </property>
    </widget>
    <addaction name="actionScan_for_software"/>
    <addaction name="menuScan_in_drive"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_all_software"/>
    <addaction name="separator"/>
    <addaction name="actionHide_software_without_path"/>
    <addaction name="actionHide_software_without_license"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionScan_for_software">
   <property name="text">
    <string>Scan all drives</string>
   </property>
  </action>
  <action name="actionShow_all_software">
   <property name="text">
    <string>Show all software</string>
   </property>
  </action>
  <action name="actionHide_software_without_path">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide software without path</string>
   </property>
  </action>
  <action name="actionHide_software_without_license">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide software without license</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
